deploy/helm/kubecf/assets/operations/instance_groups/bits.yaml

```diff
@@ -50,13 +50,14 @@
   value: local
 
 # Attach a persistent disk to bits-service VM to store eirinifs
-# TODO: storage class type should be configurable
-# - type: replace
-#   path: /instance_groups/name=bits/persistent_disk_type?
-#   value: ???
 - type: replace
   path: /instance_groups/name=bits/persistent_disk?
   value: 20480 # 20GB
+{{ if .Values.kube.storage_class }}
```

Comment:
[very nit] Use `{{- if ...}}` and `{{- end }}` to avoid a blank line in the rendered template.

Reply:
done